dvc/repo/checkout.py

```diff
@@ -10,10 +10,12 @@
 
 
 def _cleanup_unused_links(self, all_stages):
-    used = []
-    for stage in all_stages:
-        for out in stage.outs:
-            used.append(out.path)
+    used = [
+        out.fspath
+        for stage in all_stages
+        for out in stage.outs
+        if out.scheme == "local"
```

Comment:
what about SSH, aren't we optimizing it via state? cc @efiop

Reply:
As far as I can see we only use state in `RemoteLOCAL` now.